docs/content/graphql/connectors.md

```diff
@@ -0,0 +1,420 @@
+---
+title: Connectors
+order: 6
+---
+
+A connector is a piece of code that runs externally and connects foreign technologies to the connctd platform.
+Connectors are responsible for creating things and executing actions related to the things to update their state.
+Connectors can be used in apps and installed via the [Developer Center](https://devcenter.connctd.io/).
+For a deeper understanding of the connector concept and how to implement connectors, take a look at the [connector documentation](/connector/connectors/).
+Here we will focus on how connectors can be managed and used via our GraphQL API.
```

Comment:
I see your point (and that's why i did not document connectorStoreItems), but during connector development i actually preferred to use the API instead of the DevCenter, since it's way faster. I guess other developers will feel the same and i think we should make connector development as painless as possible.
I've added a few more words on why you would want to use the API.

Reply:
OK lets keep it. But I guess what was actually missing for you was some kind of a testing tool since creating/deleting installations over and over again for testing purposes is probably no enjoyable task and takes time. A simple testing tool which sends installtion/instantiation requests against your connector would have solved it. Btw ngrok has a nice replay-request function

Reply:
The connector publication combined with changing ngrok URLs was the pain point for me. Using Postman + API worked fine. We could think about publishing a Postman collection and environment with the required requests

Reply:
Sounds like the missing edit-publication-feature was the problem ;) Regarding the postman collection: I guess docs as a starting point is fine